charts/octopod/templates/octopod-nginx-configmap.yaml

```diff
@@ -16,5 +16,10 @@ data:
     {
       "app_url": "https://{{ .Values.app_domain }}",
       "ws_url": "wss://{{ .Values.ws_domain }}",
-      "app_auth": "Basic {{ .Values.basic_auth_token }}"
+      "app_auth": "Basic {{ .Values.basic_auth_token }}",
```

Comment:
JSON doesn't support tail comma

Reply:
There is an `else` clause for the `if`. This comma is never a tail comma.

Reply:
exactly.